packages/portal/src/pages/index.vue

```diff
@@ -111,6 +111,8 @@
       } else {
         this.$error(404, { scope: 'page' });
       }
+
+      this.page = Object.freeze(this.page);
```

Comment:
Does this do anything still?

Reply:
Just a general optimisation on that large data object to prevent Vue from making the whole thing reactive.